### ISSUE TITLE
Pressure fitting

### ISSUE THREAD

**Issue by MJCliffe** [MJCliffe/PASCal#27]
- [X ] bug report -> please search issues before submitting

### Minimal steps to reproduce
Some pressure datasets fail with an internal server error 500.

### Expected/desired behavior
Completing fit successfully.

Appears to be too many function calls in the fitting empirical equation. Increased maxfev at the moment but a more elegant solution is likely required.

**Comment by ml-evs** [comment]
I think I promised you a PR that commits some failing test data, will do that now.

**Comment by MJCliffe** [comment]
I think I've sorted the main issues which were:
- too small number of maxfev (to minimize compute use, but ultimately stopped reasonable data too)
- lack of bounds allowing things to underflow/overflow
- no jacobian implemented for BM2 and BM3 fitting (probably increases fevs and the chance of over/underflow too)

There's no jacobian for BM3_PC and empirical at the moment, because it's a faff to do and things are ticking over ok. I've checked against failing data provided by users and my own hard datasets and all run ok. If this reoccurs, we can try harder.
